### PR TITLE
Markdown card: add screenshot showing QR code

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Proposed change

Markdown card: add screenshot showing QR code
- inspired by <https://github.com/home-assistant/home-assistant.io/pull/34981#pullrequestreview-2337821578>

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_dashboards/markdown.markdown [hunk 1]
```diff
@@ -155,12 +154,18 @@ content: |
 
 You can also create QR-Codes in the Markdown card.
 
+<p class='img'>
+<img src='/images/dashboards/markdown_card_qr_code.png' alt='Screenshot of the markdown card with QR codes'>
```
Comment (frenck) [inline source/_dashboards/markdown.markdown:158]: Image is missing?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_dashboards/markdown.markdown (hunk 1 above)
Comment (frenck) [reply source/_dashboards/markdown.markdown:158]: I've seen you pushed something, but it seems not to be in the images folder?

CleanShot 2024-09-30 at 21 08 37@2x

It points to:
`/images/dashboards/markdown_card_qr_code.png`

but the image is in:
`source/dashboards/markdown_card_qr_code.png`

Comment (c0ffeeca7) [reply source/_dashboards/markdown.markdown:158]: :see_no_evil: 
Thank you for the catch :pray:

**Review by frenck — APPROVED** [review]
Thanks, @c0ffeeca7 👍 

../Frenck

**Pull request merged**